package.json

```diff
@@ -320,9 +323,6 @@
     "webpack": "^5.74.0",
     "webpack-cli": "^4.10.0"
   },
-  "dependencies": {
-    "markdown-it-container": "^3.0.0"
```

Comment:
do we no longer need this?

Reply:
This has been moved to the clients package.json